pydis_site/apps/api/serializers.py

```diff
@@ -679,7 +679,9 @@ class Meta:
         """Metadata defined for the Django REST Framework."""
 
         model = User
-        fields = ('id', 'name', 'discriminator', 'roles', 'in_guild')
+        fields = ('id', 'name', 'discriminator', 'roles', 'in_guild', 'alts')
+        # This should be edited on a separate endpoint only
+        frozen_fields = ('alts',)
```

Comment:
Why are we making this design decision over allowing direct updates on the existing users endpoint?

Reply:
As a client the bot would need to fetch the user, edit the list of alts, then send back the updated list to site.

Since that's not an atomic change, we could lose data if two actions were preformed on the same user at the same time.

This approach means that each action is preformed atomically by site

Reply:
Can we not support a partial update? We only need to update the alts field, not the entire user.

Reply:
Strictly speaking this line doesn't do anything at all, because from my
testing DRF already doesn't allow editing the field over the regular
patch endpoint - it just ignores whatever you give it. The
`frozen_fields` therefore serve more of a documentary purpose than
actually changing anything. I haven't found a way to enable this in
DRF...